### PR TITLE
Dat 2770 - add styling support for portable infoboxes in mercury

### PR DESCRIPTION
**Pull request by RafLeszczynski**

### CODE REVIEW

File: front/scripts/main/views/ArticleView.ts [hunk 1]
```diff
@@ -159,6 +160,39 @@ App.ArticleView = Em.View.extend(App.AdsMixin, {
...
+			minimumHeight = 450,
+			$infoboxes = $('.portable-infobox'),
+			body = window.document.body,
+			scrollTo = body.scrollIntoViewIfNeeded || body.scrollIntoView,
+			expandButton = '<div class=' + expandButtonClass + '><svg viewBox="0 0 12 7" class="icon"><use xlink:href="#chevron"></use></svg></div>'
+
+		if ($infoboxes.length) {
+			$infoboxes
+				.filter(function (index, element) {
+					console.log(element);
```
Comment (aquilax) [inline front/scripts/main/views/ArticleView.ts]: Hmmmm :dart:

Comment (RafLeszczynski) [inline front/scripts/main/views/ArticleView.ts]: ups :-)

Comment (hakubo) [inline front/scripts/main/views/ArticleView.ts]: Hey Ev,

How are you ? :)

File: front/scripts/main/views/ArticleView.ts [hunk 2]
```diff
@@ -159,6 +160,38 @@ App.ArticleView = Em.View.extend(App.AdsMixin, {
...
 
+	/**
+	 * @desc handles expanding portable infoboxes
+	 * @todo we should figure out if we can somehow merge this method and handleInfoboxes method
+	 */
+	handlePortableInfoboxes: function () {
+		var collapsedClass = 'collapsed',
+			expandButtonClass = 'portable-infobox-expand-button',
+			minimumHeight = 450,
+			$infoboxes = $('.portable-infobox'),
```
Comment (hakubo) [inline front/scripts/main/views/ArticleView.ts]: this.$('.portable-infobox') -  will scope the search to article only

File: front/scripts/main/views/ArticleView.ts [hunk 3]
```diff
@@ -159,6 +160,38 @@ App.ArticleView = Em.View.extend(App.AdsMixin, {
...
+			body = window.document.body,
+			scrollTo = body.scrollIntoViewIfNeeded || body.scrollIntoView,
+			expandButton = '<div class=' + expandButtonClass + '><svg viewBox="0 0 12 7" class="icon"><use xlink:href="#chevron"></use></svg></div>'
```
Comment (hakubo) [inline front/scripts/main/views/ArticleView.ts]: You can use template string in TS
`<div class="${expandButtonClass}"><svg viewBox="0 0 12 7" class="icon"><use xlink:href="#chevron"></use></svg></div>`

http://blogs.msdn.com/b/typescript/archive/2015/01/16/announcing-typescript-1-4.aspx

File: front/scripts/main/views/ArticleView.ts [hunk 4]
```diff
@@ -159,6 +160,38 @@ App.ArticleView = Em.View.extend(App.AdsMixin, {
...
+
+		if ($infoboxes.length) {
+			$infoboxes
+				.filter(function (index, element) {
```
Comment (hakubo) [inline front/scripts/main/views/ArticleView.ts]:
```
.filter((index, element) => $(element).outerHeight() > minimumHeight)
```

File: front/scripts/main/views/ArticleView.ts (hunk 1 above)
Comment (aquilax) [inline front/scripts/main/views/ArticleView.ts]: @hakubo I'm fine. Doing some code reviewing from time to time. I see all is good here :)

File: front/styles/main/module/article/_portable-infobox.scss [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+	}
+
+	.portable-infobox-comparison {
+		border-collapse: collapse;
+		margin: 0;
+		width: 100%;
+	}
+
+	.portable-infobox-comparison-set {
+		display: flex;
```
Comment (hakubo) [inline front/styles/main/module/article/_portable-infobox.scss]: we use autoprefixer in here
so display: flex should be enough

unless it has problems with this.

**Comment by hakubo** [comment]
lgtm

File: front/scripts/main/views/ArticleView.ts (hunk 1 above)
Comment (kenkouot) [inline front/scripts/main/views/ArticleView.ts]: LOL Hi @aquilax :D

**Pull request merged**